frontend/src/components/Board/Board.tsx

```diff
@@ -1,7 +1,327 @@
+    board_version: 1,
+    index: 5,
+    name: "Done",
+    cards: [
+      {
+        id: "3",
+        list_id: "5",
+        index: 1,
+        user_creator: "1",
+        user_assigned: "1",
+        title: "Watch Seminar Recordings",
+        description:
+          "Watch the Seminar recordings before meeting with the team.",
+        deadline: new Date().toDateString(),
+        created_at: new Date().toDateString(),
+      },
+    ],
+    created_at: new Date().toDateString(),
+  },
+];
+
+// const cards = [
+//   {
+//     id: "1",
+//     list_id: "1",
+//     index: 1,
+//     user_creator: "1",
+//     user_assigned: undefined,
+//     title: "Design the UI!",
+//     description:
+//       "Create a figma design of the initial UI to demo to the clients.",
+//     deadline: undefined,
+//     created_at: new Date().toDateString(),
+//   },
+//   {
+//     id: "2",
+//     list_id: "2",
+//     index: 2,
+//     user_creator: "1",
+//     user_assigned: "1",
+//     title: "Get to know the team!",
+//     description: "Do ice breaker activities to understand the team better.",
+//     deadline: new Date().toDateString(),
+//     created_at: new Date().toDateString(),
+//   },
+// ];
+
+const team = {
+  id: "123123",
+  board_id: "123456789",
+  user_id: "1",
+  created_at: new Date().getDate(),
+};
+
+// user_team table
+const members = [
+  {
+    id: "1",
+    email: "kev@example.com",
+    phone: "123123123",
+    raw_user_meta_data: {
+      username: "Kev",
+      given_name: "Kevin",
+      surname: "L",
+    }, // currently how supabase by default wants to store user data
+  },
+  {
+    id: "2",
+    email: "mursal@example.com",
+    phone: "123123123",
+    raw_user_meta_data: {
+      username: "Mursal",
+      given_name: "Mursal",
+      surname: "A",
+    },
+  },
+];
+
+interface IColumn {
+  id: string;
+  board_id: string;
+  board_version: number;
+  index: number;
+  name: string;
+  cards: ICard[];
+  created_at: string;
+}
+
+interface ICard {
+  id: string;
+  list_id: string;
+  index: number;
+  user_creator: string;
+  user_assigned: string | undefined;
+  title: string;
+  description: string;
+  deadline: string | undefined;
+  created_at: string;
+}
+
+interface IColumnProp {
+  column: IColumn;
+}
+
+function TaskCard(props: { item: ICard }) {
+  const item = props.item;
+  // console.log(item)
+  return (
+    <>
+      <Card className="m-1" sx={{ maxWidth: 345 }}>
+        <CardContent>
+          <Typography gutterBottom variant="h6" component="div">
+            {item.title}
+          </Typography>
+          <Typography variant="body2" color="text.secondary">
+            {item.description}
+          </Typography>
+        </CardContent>
+        <CardActions className="flex flex-col">
+          <Button className="justify-self-end" size="small">
+            Edit
+          </Button>
+        </CardActions>
+      </Card>
+    </>
+  );
+}
+
+function TaskColumn(props: IColumnProp) {
```

Comment:
lets try and directly destructure props, cleaner code, avoids repetitively calling "props." everywhere

Reply:
yea, my bad, i initially had alot more stuff in that props, but I removed them later on.

Reply:
opps nvm, i read ur comment wrong, ill do that